src/api-explorer/v4-0/FinancialIntegrationService.swagger2.json

```diff
@@ -0,0 +1,867 @@
+            "required": false,
+            "type": "string",
+            "enum": [
+              "yes",
+              "no"
+            ]
+          },
+          {
+            "name": "systemId",
+            "in": "query",
+            "description": "systemId",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "Authorization",
+            "in": "header",
+            "description": "For details on how to get an authorization token, please refer <a href=\"https://developer.concur.com/api-reference/authentication/getting-started.html\"> here </a>",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PagedResources.Resource.FIDocument"
+            }
+          },
+          "400": {
+            "description": "Invalid request"
+          },
+          "500": {
+            "description": "Internal error",
+            "schema": {
+              "$ref": ""
+            }
+          }
+        }
+      }
+    },
+    "/fi/v4/companies/transactiontypes/{docType}/transactions/acknowledgements": {
+      "post": {
+        "tags": [
+          "Financial Transactions"
+        ],
+        "summary": "Acknowledges successful retrieval of financial transaction(s).",
+        "description": "This API allows a financial system to acknowledge/confirm that it has successfully retrieved one or more financial transactions and will begin to process those transactions. These transactions are then taken out of the ready queue.",
+        "operationId": "acknowledgeUsingPOST",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "docType",
+            "in": "path",
+            "description": "Document type",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "expense",
+              "invoice",
+              "cashadvance",
+              "payroll",
+              "obligation"
+            ]
+          },
+          {
+            "in": "body",
+            "name": "acknowledgeRequest",
+            "description": "Acknowledge Request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AcknowledgeRequest"
+            }
+          },
+          {
+            "name": "Authorization",
+            "in": "header",
+            "description": "Bearer JWTFromOAuthServiceHere",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/AcknowledgeResponseItem"
```

Comment:
The markdown lists an item called AcknowledgeResponse, an array that contains multiple AcknowledgeResponseItem objects. The swagger only lists AcknowledgeResponseItem, which is correct?